schema_utils.py

```diff
@@ -229,47 +259,118 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to evaluate.
+            min_value: int. Minimum value of elements.
+
+        Returns:
+            bool. True if the number of elements is greater than or equal
+                to min_value.
         """
         return len(obj) >= min_value
 
     @staticmethod
     def has_length_at_most(obj, max_value):
         """Returns True iff the given object (a list) has at most
         `max_value` elements.
+
+        Args:
+            obj: *. Object to evaluate.
+            max_value: int. Maximum value of elements.
+
+        Returns:
+            bool. True if the number of elements is less than or equal
+                to min_value.
```

Comment:
There is no min_value being passed in here.

Reply:
Done